docker-compose.yml

```diff
@@ -29,6 +29,9 @@ services:
     profiles: ["db"]
```

Comment:
remove this file changes not related to the PR

Reply:
ah yes